docs/EXTENDED-ARCHIVES.md

```diff
@@ -201,6 +212,15 @@ and `md5`.
 
 See the `.meta/checksum` file.
 
+##### `path (String)`
+
+The path to the image, relative to the root of the archive.
```

Comment:
Possibly worth highlighting that this has a different relative path to the other files referenced here (e.g. by making it bold?)